router/router.go

```diff
@@ -304,6 +304,7 @@ func SupportCORS(handler http.Handler) http.Handler {
 		AllowOriginFunc: func(string) bool {
 			return true
 		},
+		AllowedOrigins: []string{"*", "https://prebid.adnxs.com/pbs/v1/cookie_sync"},
 		AllowedHeaders: []string{"Origin", "X-Requested-With", "Content-Type", "Accept"}})
```

Comment:
According to the CORS documentation found in golang's CORS API itself:
```
// Options is a configuration container to setup the CORS middleware.
type Options struct {
	// AllowedOrigins is a list of origins a cross-domain request can be executed from.
	// If the special "*" value is present in the list, all origins will be allowed.
	// An origin may contain a wildcard (*) to replace 0 or more characters
	// (i.e.: http://*.domain.com). Usage of wildcards implies a small performance penalty.
	// Only one wildcard can be used per origin.
	// Default value is ["*"]
	AllowedOrigins []string
vendor/github.com/rs/cors/cors.go
```
Since the default value is supposed to be `"*"`, I figured we could add `https://acdn.adnxs.com` explicitly while also keeping the `"*"` value. I believe this solution should work but, it is better to test it first.